### PR TITLE
[BUG] \Phalcon\Session\Flash::getMessages() #11941

### PR DESCRIPTION
**Pull request by tugrul**
#11941

### CODE REVIEW

File: phalcon/flash/session.zep [hunk 1]
```diff
@@ -67,7 +67,11 @@ class Session extends FlashBase implements FlashInterface, InjectionAwareInterfa
 		let session = <SessionInterface> dependencyInjector->getShared("session");
 		let messages = session->get("_flashMessages");
 
-		if typeof type == "string" && isset(messages[type]) {
+		if typeof type == "string" {
+			if !isset(messages[type])) {
+				return [];
+			}
+			
 			if !fetch returnMessages, messages[type] {
```
Comment (Jurigag) [inline phalcon/flash/session.zep]: Isn't this fetch supposed to do what this if above is doing ?

Comment (tugrul) [inline phalcon/flash/session.zep]: @Jurigag I'm not familiar to code Zephir. I looked up behavior of `fetch` operator and you are right.

Following block possibly better:
```zephir
if typeof type == "string" {
    if fetch returnMessages, messages[type] {
        if remove === true {
            unset(messages[type]);
            session->set("_flashMessages", messages);
        }
        return returnMessages;
    }
    
    return [];
}
```

I'm going to commit this.

**Comment by sergeyklay** [comment]
We don't add changes directly to stable branches or `master`. Could you please send this PR to the `2.1.x` branch?

**Comment by sergeyklay** [comment]
Fixed in the `2.1.x` branch.

**Pull request closed without merging**